src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -74,18 +74,17 @@ internal static TypeSymbol CopyTypeCustomModifiers(TypeSymbol sourceType, TypeSy
             // code gen uses and then passing them to the dynamic type decoder that metadata reading uses.
```

Comment:
nit: Consider revising the `<param name="destinationType">` comment on line 63 to delete its last sentence "*May differ in object/dynamic.*", since there are several other scenarios that this method handles, possibly more in the future, and we don't want to give an impression that object/dynamic is all that we're handling here.

Reply:
I'll update the comment in a subsequent PR.

---
In reply to: [490547894](https://github.com/dotnet/roslyn/pull/47536#discussion_r490547894) [](ancestors = 490547894)